.github/workflows/create-landing-page.yml

```diff
@@ -0,0 +1,39 @@
+name: create-landing-page
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'CITATION.cff'
+  release:
+    types: [published]
+
+jobs:
+  update-landing-page:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Update pip and install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install cffconvert==1.3.3
+      - name: Set env var based on (master) push trigger
+        if: github.event_name == 'push'
+        run: echo "PAGE_TYPE=latest" >> $GITHUB_ENV
+      - name: Set env var based on release
+        if: github.event_name == 'release'
+        run: echo "PAGE_TYPE=${{ github.event.release.name }}" >> $GITHUB_ENV
+      - name: Create landing page
+        run: python write_doi_landing_page.py ${{ env.PAGE_TYPE }}
```

Comment:
Might it be better to have this python script (`write_do_landing_page.py`) in the main branch rather than the `gh-pages` branch? (so that changes to it are more transparently seen...?) This would mean a weird checkout of main and then a checkout of `gh-pages` though...

Reply:
I feel like it does belong in the gh-pages branch so that it doesn't clutter up the repo that most people will check out (as it's literally only for creating the website), and if it's there we can make changes to how the website is laid out if we want without having to make commits/PRs to master